src/controllers/admin.controller.js

```diff
@@ -63,36 +81,4 @@ adminCtrl.deleteStory = (req, res) => {
     res.redirect("/admin/Story");
 };
 
-// AT-RESOURCES - Admin - Render Edit Story Form
-adminCtrl.renderEditStoryForm = async(req, res) => {
-    console.log("--> adminCtrl.renderEditStoryForm");
-    res.render("admin/story/edit-story");
-}
-// AT-RESOURCES - Admin - Delete Story
-adminCtrl.deleteStory = async(req, res) => {
```

Comment:
Why did you remove this function?

Reply:
it is duplicated:
https://github.com/at-internship/at-resources-ui/commit/2f9c8c7535e913c24ab1f456754f17dd96e67546
But if it exists

Reply:
duplicated in lines 70,76